Shared/Tests.Shared/TestObjects.cs

```diff
@@ -44,6 +44,9 @@ public class AllTypesObject : RealmObject
 
         public DateTimeOffset DateTimeOffsetProperty { get; set; }
 
+        [Required]
+        public string RequiredStringProperty { get; set; }
```

Comment:
This doesn't seem to be used, is it just to validate that `[Required]` doesn't cause any funny behavior?